### PR TITLE
Portal

### PR DESCRIPTION
**Pull request by sjpadgett**
- needed larger container for some templates

### CODE REVIEW

File: sql/5_0_1-to-5_0_2_upgrade.sql [hunk 1]
```diff
@@ -633,6 +633,10 @@ ALTER TABLE `users_secure` ADD COLUMN `login_work_area` text;
 ALTER TABLE `onsite_messages` CHANGE `sender_id` `sender_id` VARCHAR(64) NULL COMMENT 'who sent id';
 #EndIf
 
+#IfNotColumnType onsite_documents full_document MEDIUMBLOB
+ALTER TABLE `onsite_documents` CHANGE `full_document` `full_document` MEDIUMBLOB NULL DEFAULT NULL;
```
Comment (bradymiller) [inline sql/5_0_1-to-5_0_2_upgrade.sql]: hi @sjpadgett ,
remove the `NULL DEFAULT NULL`

Comment (bradymiller) [reply sql/5_0_1-to-5_0_2_upgrade.sql]: Also should put this at the bottom of the script (we keep it chronological so we don't go mad if need to figure things out)

**Comment by bradymiller** [comment]
One more thing, when make a database change in upgrade script, need to increment this field:
https://github.com/openemr/openemr/blob/master/version.php#L28

**Comment by sjpadgett** [comment]
@bradymiller Just realized i'm going to need to add a db patch for my back ports. Can I just add to 5.0.1 branch ie. patch.sql then roll it into a formal patch?

**Comment by sjpadgett** [comment]
@bradymiller I get it. You can't do it all so this can wait. Besides, i'm going to stop doing back ports as v5.0.2 has left v5.0.1 behind, left the station so to say. Or we could just encourage folks from developing anymore stuff. I'll see if I can muddle a patch together. May not be as eloquent as yours but, I like folks to have to guess what's in a patch:)

**Pull request merged**